packages/core/package.json

```diff
@@ -44,14 +44,11 @@
     "b64-lite": "1.4.0",
     "bad-behavior": "1.0.1",
     "cookie-session": "1.4.0",
-    "deepmerge": "4.2.2",
     "lodash": "^4.17.19",
-    "lodash-es": "^4.17.15",
```

Comment:
@KATT we have to keep `lodash-es` according to [tsdx lodash instructions](https://tsdx.io/optimization#using-lodash)